packages/ckeditor5-mention/src/mentionui.ts

```diff
@@ -821,29 +761,85 @@ function isPositionInExistingMention( position ) {
 	return hasMention || nodeBefore && nodeBefore.is( '$text' ) && nodeBefore.hasAttribute( 'mention' );
 }
 
-// Checks if the closest marker offset is at the beginning of a mention.
-//
-// See https://github.com/ckeditor/ckeditor5/issues/11400.
-//
-// @param {module:engine/model/position~Position} markerPosition
-// @returns {Boolean}
-function isMarkerInExistingMention( markerPosition ) {
+/**
+ * Checks if the closest marker offset is at the beginning of a mention.
+ *
+ * See https://github.com/ckeditor/ckeditor5/issues/11400.
+ */
+function isMarkerInExistingMention( markerPosition: Position ): boolean | null {
 	const nodeAfter = markerPosition.nodeAfter;
 
 	return nodeAfter && nodeAfter.is( '$text' ) && nodeAfter.hasAttribute( 'mention' );
 }
 
-// Checks if string is a valid mention marker.
-//
-// @param {String} marker
-// @returns {Boolean}
-function isValidMentionMarker( marker ) {
+/**
+ * Checks if string is a valid mention marker.
+ */
+function isValidMentionMarker( marker: string ): boolean | string {
 	return marker && marker.length == 1;
 }
 
-// Checks the mention plugins is in completion mode (e.g. when typing is after a valid mention string like @foo).
-//
-// @returns {Boolean}
-function checkIfStillInCompletionMode( editor ) {
+/**
+ * Checks the mention plugins is in completion mode (e.g. when typing is after a valid mention string like @foo).
+ */
+function checkIfStillInCompletionMode( editor: Editor ): boolean {
 	return editor.model.markers.has( 'mention' );
 }
+
+/**
+ * Fired whenever requested feed has a response.
```

Comment:
Docs for params are misleading - there is only one param which is an object. Create a separate type for this object to have a place to attach those docs.